Source/UI/NewUI/BillingForm/Factory/BillingFormFactory.swift

```diff
@@ -18,16 +18,19 @@ public struct BillingFormFactory {
          .phoneNumber(DefaultBillingFormPhoneNumberCellStyle())]
     }
 
+    static func getBillingFormViewController(style: BillingFormStyle?, data: BillingForm?, delegate: BillingFormViewModelDelegate) -> (BillingFormViewModelDelegate?, UINavigationController?) {
```

Comment:
would this be a better representation of the return types to be `(BillingFormViewModelDelegate, UINavigationController)?` instead?

I can't see a case where one is nil and the other isn't